lib/tasks/compare_xml_collections.rake

```diff
@@ -0,0 +1,121 @@
+require 'libxml_to_hash'
+
+namespace :collection do
+  desc 'Translate a collection from native format to UMM JSON and back to native format'
+  task :loss, [:file, :format, :disp, :version] => :environment do |_task, args|
+    args.with_defaults(:version => '1.15.3')
+    args.with_defaults(:disp => 'show')
+
+    abort 'FORMAT INVALID' unless args.format.eql?('echo10') || args.format.eql?('dif10') || args.format.eql?('iso19115')
```

Comment:
made a comment about using equal signs before. still think we should be consistent. [this post](https://stackoverflow.com/questions/7156955/whats-the-difference-between-equal-eql-and) and the links might help.
Also, what's the reason these are the only formats we can use? A comment might help.

Reply:
I appreciate the link - just read through it. As for the reason for only using those formats, I am not sure what the other formats are (that cmr supports for collections). This is from the [cmr data partner's guide](https://wiki.earthdata.nasa.gov/display/CMR/CMR+Data+Partner+User+Guide) so I just assumed these were the only ones: 
<img width="487" alt="Screen Shot 2020-07-20 at 9 56 53 AM" src="https://user-images.githubusercontent.com/42478387/87945958-7026e800-ca6f-11ea-9791-97deb2ad48fa.png">